src/test/lit-html_test.ts

```diff
@@ -225,7 +225,13 @@ suite('lit-html', () => {
         assert.equal(container.innerHTML, '<div>foo bar</div>');
       });
 
-      test('renders nested templates within table content', () => {
+      const testSkipIE =
```

Comment:
Rather than a UA-sniff, I think it'd be good to check for the template polyfill. HTMLTemplateElement.decorate should be a good property to check.

Reply:
That is indeed better than a UA sniff, thanks! I've updated the PR.